lib/clevertap_plugin.dart

```diff
@@ -192,12 +196,49 @@ class CleverTapPlugin {
           cleverTapOnFileValueChangedHandler(args.cast<String, dynamic>());
         });
         break;
+      case "customTemplatePresent":
+        if (call.arguments is Map) {
```

Comment:
Will customTemplatePresent ever return a map from the native side?
Can we force it to return string from the native side instead of having a check here?

Reply:
Resolved 
No need of map parameter check condition anymore